ci/kokoro/docker/build-in-docker-bazel.sh

```diff
@@ -211,15 +147,15 @@ if [[ "${RUN_INTEGRATION_TESTS}" == "yes" || \
   echo
   echo "================================================================"
   log_normal "Create an access token to run the Bigtable credential examples."
-  "${GCLOUD}" "${GCLOUD_ARGS[@]}" auth activate-service-account --key-file \
-      "${GOOGLE_APPLICATION_CREDENTIALS}"
+  activate_service_account_keyfile "${GOOGLE_APPLICATION_CREDENTIALS}"
   # This is used in a Bigtable example showing how to use access tokens to
   # create a grpc::Credentials object. Even though the account is deactivated
   # for use by `gcloud` the token remains valid for about 1 hour.
   ACCESS_TOKEN="$("${GCLOUD}" "${GCLOUD_ARGS[@]}" auth print-access-token)"
   readonly ACCESS_TOKEN
+
   # Deactivate the recently activated service accounts to prevent accidents.
-  "${GCLOUD}" "${GCLOUD_ARGS[@]}" auth revoke --all
+  revoke_service_account_keyfile "${GOOGLE_APPLICATION_CREDENTIALS}"
```

Comment:
Perhaps this one now belongs in the `EXIT` handler too?

Reply:
Done.